frontend/src/api/QuestionApi.js

```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
```

Comment:
standardize to by using const ... = require('..''_

Reply:
Issue encountered here